js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,6 +1,6 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js');
+var ExtensionStore = require('./ExtensionStore.js').instance;
```

Comment:
How come we have a .instance property, rather than just exporting a single object, or a .getInstance() helper if we have a singleton but need some future-proofing?